public/application.js

```diff
@@ -1,9 +1,36 @@
 $(document).ready(function(){
+  
   var name = $('#name');
   var email = $('#email');  
+  var username = $('#username');
+  var password = $('#password');  
+  
   var username2 = $('#username2');
   var password2 = $('#password2');  
-  
+
+  // $('#sign-in').click(function() {
```

Comment:
My Sign-up  function wasn't working at the time so I commented out the sign-in to try to isolate the issue. Resolved now.